packages/webdriver/protocol/appium.json

```
@@ -944,10 +944,14 @@
       "description": "Set GSM signal strength (Emulator only).",
       "ref": "http://appium.io/docs/en/commands/device/network/gsm-signal/",
       "parameters": [{
-        "name": "signalStrengh",
+        "name": "signalStrength",
         "type": "string",
-        "description": "signal strength in the range [0, 4]",
+        "description": "signal strength in the range [0, 4]. This parameter works with Appium 1.11.1 upwards.",
         "required": true
+      }, {
+        "name": "signalStrengh",
+        "type": "string",
+        "description": "signal strength in the range [0, 4]"
```

```suggestion
        "description": "signal strength in the range [0, 4]. Please also set this parameter with the same value if you use Appium v1.11.0 or lower (see https://github.com/appium/appium/issues/12234)."
```

Let's be 100% sure and add this comment.

---

Hi @christian-bromann The message now is exactly as your expectation.

---

@khanhdodang this description should be for the `signalStrengh` parameter .. the other can stay at it was before

---

Hi @christian-bromann I updated